src/components/ToolBar/DataMenu/UploadNetworkMenuItem.tsx

```diff
@@ -82,20 +91,43 @@ export const UploadNetworkMenuItem = (props: BaseMenuProps): ReactElement => {
     reader.onload = async (event) => {
       try {
         const json = JSON.parse(event.target?.result as string)
-        let localName: string = "";
-          for (const item of json) {
-            if ((Boolean(item.networkAttributes)) && typeof item.networkAttributes[0].name === 'string') {
-              localName = item.networkAttributes[0].name;
-              break;
-            }
+        let localName: string = ''
+        for (const item of json) {
+          if (
+            Boolean(item.networkAttributes) &&
+            typeof item.networkAttributes[0].name === 'string'
+          ) {
+            localName = item.networkAttributes[0].name
+            break
+          }
         }
-        let localDescription: string = "";
+        let localDescription: string = ''
         for (const item of json) {
-          if ((Boolean(item.networkAttributes)) && typeof item.networkAttributes[0].name === 'string') {
-            localDescription = item.networkAttributes[0].description;
-            break;
+          if (
+            Boolean(item.networkAttributes) &&
+            typeof item.networkAttributes[0].name === 'string'
+          ) {
+            localDescription = item.networkAttributes[0].description
+            break
           }
         }
+        const networkAttributeDeclarations =
+          getAttributeDeclarations(json).attributeDeclarations[0]
+            .networkAttributes
+        const networkAttributes = getNetworkAttributes(json)[0]
```

Comment:
`getAttributeDeclarations` and `getNetworkAttributes` are util functions from `models/CxModel/cx2-util.ts`.  These allow you to get various aspects of the cx and merge them to create proper `NdexNetworkProperty` objects for the summary.